svc.html

```diff
@@ -0,0 +1,347 @@
+              full framerate, 2.0 = one half of the full framerate. For scalable video
+              coding, <code>scaleFramerateDownBy</code> refers to the inverse of the aggregate
+              fraction of input framerate achieved by this layer when combined with all
+              dependent layers.</p>
+            </dd>
+            <dt><dfn data-idl><code>encodingId</code></dfn> of type <span class=
+            "idlMemberType"><a>DOMString</a></span></dt>
+            <dd>
+              <p>An identifier for the encoding object. This identifier should be unique
+              within the scope of the localized sequence of
+              <code><a>RTCRtpEncodingParameters</a></code> for any given
+              <code><a>RTCRtpParameters</a></code> object. Values MUST be composed only
+              of alphanumeric characters (a-z, A-Z, 0-9) up to a maximum
+              of 16 characters. While the <code>encodingId</code> attribute has the same syntax
+              as the <code>rid</code> attribute, these attributes are distinct, since the
+              <code>encodingId</code> attribute does not imply inclusion of a RID header extension
+              within RTP packets.</p>
+            </dd>
+            <dt><dfn data-idl><code>dependencyEncodingIds</code></dfn> of type <span class=
+            "idlMemberType">sequence&lt;<a>DOMString</a>&gt;</span></dt>
+            <dd>
+              <p>The <code><a>encodingId</a></code>s on which this layer depends. Within
+              this specification <code><a>encodingId</a></code>s are permitted only
+              within the same <code><a>RTCRtpEncodingParameters</a></code> sequence.
+              In order to send scalable video coding
+              (<a>SVC</a>), both the <code>encodingId</code> and
+              <code>dependencyEncodingIds</code> are required.</p>
+            </dd>
+          </dl>
+        </section>
+      </div>
+    </section>
+    </section>
+    <section id="rtcrtpencodingparameters-example*">
+      <h3>Examples</h3>
+        <section class="informative" id="rtcrtpencodingtemporal-example*">
+          <h4>Temporal Scalability</h4>
+          <pre class="example highlight">
+// Example of 3-layer temporal scalability encoding with base framerate
+// one quarter of the input, and ehancement layers providing one half
+// and all of the input framerate
+var encodings = [
+  {encodingId: 'T0', scaleFramerateDownBy: 4.0},
+  {encodingId: 'T1', scaleFramerateDownBy: 2.0, dependencyEncodingIds: ['T0']},
+  {encodingId: 'T2', dependencyEncodingIds: ['T0', 'T1']} 
+];
+
+// Example of 3-layer temporal scalability with all but the base layer disabled
+var encodings = [
+  {encodingId: 'T0', scaleFramerateDownBy: 4.0, active: true}, 
+  {encodingId: 'T1', scaleFramerateDownBy: 2.0, dependencyEncodingIds: ['T0'], active: false},
+  {encodingId: 'T2', dependencyEncodingIds: ['T0', 'T1'], active: false}
```

Comment:
What happens if I only specify a dependency on T1? Or only T0?
What happens if I create a circular dependency?
What happens if I don't specify the dependency?
What if I specify a dependency T0 -> T2 instead of T2 -> T0?
Etc.